assets/css/common/post-entry.css

```diff
@@ -40,6 +40,7 @@
     background: var(--entry);
     border-radius: var(--radius);
     transition: transform .1s;
+    transition: background-color 350ms, border 350ms;
```

Comment:
This should be changed to the following, otherwise the second rule will override the first, effectively canceling it.
```css
transition: transform .1s, background-color 350ms, border 350ms;
```

Reply:
Apologies.
I'm working on a serious bug on another project currently.
I half-mindedly added that so I could go back to the project.
Fixing it soon